Makefile

```diff
@@ -8,11 +8,21 @@ generate-values:
 	@cat common-values.yaml charts/kubewarden-defaults/chart-values.yaml >> charts/kubewarden-defaults/values.yaml
 
 .PHONY: check-generated-values
-check-generated-values: generate-values
+check-generated-values: generate-values generate-images-file generate-policies-file
 	@sh -c 'git diff --exit-code charts || (echo; echo "There are chart differences that have to be checked in"; exit 1)'
+	@sh -c 'git diff --exit-code policylist.txt || (echo; echo "There are policylist.txt differences that have to be checked in"; exit 1)'
+	@sh -c 'git diff --exit-code imagelist.txt || (echo; echo "There are imagelist.txt differences that have to be checked in"; exit 1)'
```

Comment:
I think this 2 lines aren't needed; the first `git diff` of the charts path will find the differences and stop there.

Reply:
This is not necessary anymore. My fix in the `chart-releaser` enable us to upload the files in the Github release.